package-scope.qmd

```diff
@@ -1,55 +1,43 @@
+1. We minimize dependencies used for plotting functions in our packages.
 
-Epiverse-TRACE packages are not developed uniformly and certain packages may deviate from this philosophy. In these cases the reasoning for the difference should be documented in that package's design vignette.
+2. We provide detailed plotting examples throughout our vignettes, which may rely on external dependencies and include reproducible code. 
 
-## Epiverse-TRACE reasoning
+Epiverse packages may vary from these principles on a case by case basis. Packages that do must document the motivation in the package vignette.
 
-A benefit of including plotting functions is it enables package users to quickly inspect a feature of the package (e.g. model output) without having to write their own script or function. Despite this clear benefit of shipping plotting with a package there are several
-drawbacks, and here we outline the reasons why only dependency-free plotting (also commonly referred to as base R plotting) is included in Epiverse-TRACE R packages.
+## Reasoning
 
-The most frequently used plotting package in R is [{ggplot2}](https://ggplot2.tidyverse.org/index.html). This package implements the grammar of graphics and provides users with a extensive and flexible set of tools to build custom plots. These are the reasons it is used for plotting in package vignettes. However, in
-order for an R package to export plotting functions that use {ggplot2}, it must list this package
-as a [dependency](https://cran.r-project.org/doc/manuals/R-exts.html#Package-Dependencies). {ggplot2} is a "heavy" package, not so much from the size of the code base (as CRAN limits the size of packages to 5MB), but due to it's own dependencies.
+We include dependency free plotting in our packages (approach 1) to [minimize dependency bloat](https://epiverse-trace.github.io/blueprints/dependencies.html) and maximize maintainability. We must balance the benefits of dependencies with package development. 
 
-```{r, pkg-deps, eval=FALSE}
-tools::package_dependencies(packages = "ggplot2", recursive = FALSE)
+For example, we could include the most widely used plotting package, [{ggplot2}](https://ggplot2.tidyverse.org/index.html), as a dependency.
 
-tools::package_dependencies(packages = "ggplot2", recursive = TRUE)
-```
+However, {ggplot2} includes a fair amount of its own dependencies (`r length(tools::package_dependencies(packages = "ggplot2", recursive = TRUE)$ggplot2)` to be precise), risking dependency blaot. 
 
-The relatively large number of dependencies introduces a couple of issues:
+Within the wider Epiverse, we also [saw changes to {ggplot2}](https://github.com/reconhub/incidence/issues/119), breaking package implementations. This is always a risk of dependencies and goes against our [minimal dependency guideline](https://epiverse-trace.github.io/blueprints/dependencies.html).
```

Comment:
Perhaps rephrase "Within the wider Epiverse", as {incidence} is a RECON package. Something along the lines of "In other epidemiology R packages...".